README.md

```diff
@@ -1 +1,189 @@
+2:removes silly errors before execution and code review.<br>
+3:Makes code look like written by a single person.<br>
+4:Spreads awareness and ownership over code quality<br>
+5:Having more secure and performant code.<br>
+
+## Install Linters
+
+You can find linters for most of the programming languages, e.g. Rubocop for Ruby or ESLint for JavaScript.
+
+Also, there are many ways you can integrate a linter in your workflow:
+
+-text editor plugin<br>
+-GitHub Actions<br>
+-GitHub apps<br>
+
+## Set up Linters
+
+**Note:** The npm package manager is going to create a node_modules directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a .gitignore file and add node_modules to it:
+
+# .gitignore
+
+node_modules/
+
+## Web Hint
+
+This is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
+
+**NOTE:** If you are using Windows, make sure you initialize npm to create `package.json` file.
+
+```
+npm init -y
+```
+
+1. Run
+   ```
+   npm install --save-dev hint@7.x
+   ```
+   _how to use npm: (https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)._
+2. Copy [.hintrc](.hintrc) to the root directory of your project.
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+4. Run
+   ```
+   npx hint .
+   ```
+5. Fix validation errors.
+
+### [Stylelint](https://stylelint.io/)
+
+A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
+
+1. Run
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+not sure how to use npm? Read this.
+
+2. Copy .stylelintrc.json to the root directory of your project.
+
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+
+If you think that change is necessary - open a Pull Request in this repository and let your code reviewer know about it. 4. Run npx stylelint "\*_/_.{css,scss}" on the root of your directory of your project.
+
+5. Fix linter errors.
+
+6. **IMPORTANT NOTE:** feel free to research auto-correct options for Stylelint if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: (https://github.com/siddghosh108)
+- Twitter: (https://twitter.com/siddharthaghos9)
+- LinkedIn: (https://www.linkedin.com/in/siddhartha-ghosh-65902718)
```

Comment:
- [ ] For your author link handlers, please use the same template as in the image, just remove the red-underlined parts and write your username instead depending on the platform, this will make your readme file look neat and professional.
 
 ![Professinal git authur](https://github.com/siddghosh108/Hello_World/assets/92458236/4fb6803d-3d2e-4f34-905c-c6565897794d)

Reply:
I have changed them accordingto your suggestion

- GitHub: [@siddghosh108](https://github.com/siddghosh108)
- Twitter: [@siddharthaghos9](https://twitter.com/siddharthaghos9)
- LinkedIn: [@siddhartha-ghosh-65902718](https://www.linkedin.com/in/siddhartha-ghosh-65902718)